lib/brcobranca/retorno/cnab240/base.rb

```diff
@@ -0,0 +1,37 @@
+# -*- encoding: utf-8 -*-
+require 'parseline'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.